tests/conftest.py

```diff
@@ -158,6 +158,17 @@ def multidict_getversion_callable(multidict_module: ModuleType) -> Callable:
     return multidict_module.getversion
 
 
+@pytest.fixture(params=list(range(pickle.HIGHEST_PROTOCOL + 1)))
```

Comment:
Does this work without turning the generator into a list?

Reply:
If not, please use tuple instead.

Reply:
There's already a `pickle_protocol` that is injected through `pytest_generate_tests`. Can't you depend on it instead of maintaining a copy of the same logic?